roles/splunk_common/tasks/enable_service.yml

```diff
@@ -22,8 +22,9 @@
   become: yes
   become_user: "root"
   systemd:
-      name: daemon-reload
-      state: restarted
+      daemon-reload: yes
+      name: splunk.service
```

Comment:
Is `name` required here for daemon-reload? We may possibly just want to tell systemd to re-read all configs here?